plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -71,15 +101,28 @@ export const MissingEventsRestEndpoint = () => (
 );
 
 export const isSplunkOnCallAvailable = (entity: Entity) =>
-  Boolean(entity.metadata.annotations?.[SPLUNK_ON_CALL_TEAM]);
+  Boolean(entity.metadata.annotations?.[SPLUNK_ON_CALL_TEAM]) ||
+  Boolean(entity.metadata.annotations?.[SPLUNK_ON_CALL_ROUTING_KEY]);
+
+const useStyles = makeStyles({
+  onCallCard: {
+    marginBottom: '1em',
+  },
+});
 
 export const EntitySplunkOnCallCard = () => {
+  const classes = useStyles();
   const config = useApi(configApiRef);
   const api = useApi(splunkOnCallApiRef);
   const { entity } = useEntity();
   const [showDialog, setShowDialog] = useState<boolean>(false);
   const [refreshIncidents, setRefreshIncidents] = useState<boolean>(false);
-  const team = entity.metadata.annotations![SPLUNK_ON_CALL_TEAM];
+  const teamAnnotation = entity
+    ? entity.metadata.annotations![SPLUNK_ON_CALL_TEAM]
+    : undefined;
+  const routingKeyAnnotation = entity
+    ? entity.metadata.annotations![SPLUNK_ON_CALL_ROUTING_KEY]
+    : undefined;
```

Comment:
☝️ I added the ternary logic checking for `undefined` to aid in local dev, where running `yarn start` from within `plugins/splunk-on-call` otherwise results in entity not found errors. Perhaps this is misguided?